pages/docs/tutorials/javascript/setting-up.md

```diff
@@ -1,106 +1,154 @@
-
-When targeting other platforms alongside JavaScript, the Multiplatform plugin can be used instead of the JS plugin. An empty template for such a project can be created through the wizard in IntelliJ IDEA:
-
-![Multiplatform project wizard]({{ url_for('tutorial_img', filename='javascript/setting-up/multiplatform-project.png')}})
-
-After creating a multiplatform plugin with the wizard, any kind of platform-specific configuration is omitted at first. To add the JavaScript target, we adjust our automatically generated `build.gradle.kts` file to look analogous to this:
-
-<div class="sample" markdown="1" theme="idea" data-highlight-only>
-```kotlin
-plugins {
-    kotlin("multiplatform") version "1.3.70"
-}
-
-group = "org.example"
-version = "1.0-SNAPSHOT"
-
-repositories {
-    mavenCentral()
-}
-
-kotlin {
-    js {
-        browser { }
-    }
-
-    sourceSets {
-        val commonMain by getting {
-            dependencies {
-                implementation(kotlin("stdlib-common"))
-            }
-        }
-        val commonTest by getting {
-            dependencies {
-                implementation(kotlin("test-common"))
-                implementation(kotlin("test-annotations-common"))
-            }
-        }
-
-        val jsMain by getting {
-            dependencies {
-                implementation(kotlin("stdlib-js"))
-            }
-        }
-
-        val jsTest by getting {
-            dependencies {
-                implementation(kotlin("test-js"))
-            }
+2. Change the code of `styledDiv` to show your name backwards.  
+   
+    * Use the standard library function `reversed()` to reverse your name.
+    * Use a [string template](https://kotlinlang.org/docs/reference/basic-types.html#string-templates) for your reversed 
+    name by adding a dollar sign `$` and enclosing it in curly braces – `${state.name.reversed()}`.
+    
+    <div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+    
+    ```kotlin
+    styledDiv {
+        css {
+            +WelcomeStyles.textContainer
         }
+        +"Hello ${state.name}!"
+        +" Your name backwards is ${state.name.reversed()}!"
     }
-}
-```
-</div>
+    ```
+    
+    </div>
+
+3. Save your changes to the file.
+
+4. Go to the browser and enjoy the result.  
+    You will see the changes only if your previous application is still running. If you've stopped your application, [run it again](#run-the-application).
+   
+<img class="img-responsive" src="{{ url_for('tutorial_img', filename='javascript/setting-up/js-output-2.png') }}" alt="Web browser with a reversed name" />
+
+### Add an image
+
+1. Open the file `welcome.kt` in `src/main/kotlin`.  
+
+2. Add a `div` container with a child image element `img` after the `styledInput` block.  
+   
+    > Make sure that you import the `react.dom.*` and `styled.*` packages.
+    {:.note}       
+    
+    <div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
```

Comment:
Somehow, this renders as an indentation with 5 spaces instead of four. It also means the last line ends up with an extra space, for some reason, which shouldn't be there:
![image](https://user-images.githubusercontent.com/2178959/88584719-5f1f3d80-d052-11ea-9b67-a2ae94795d11.png)

Reply:
🛠